src/main/java/edu/harvard/iq/dataverse/search/SearchServiceBean.java

```diff
@@ -741,23 +744,23 @@ public SolrQueryResponse search(
 //                logger.info("field: " + facetField.getName() + " " + facetFieldCount.getName() + " (" + facetFieldCount.getCount() + ")");
                 String localefriendlyName = null;
                 if (facetFieldCount.getCount() > 0) {
-                   if(metadataBlockName.length() > 0 ) {
-                       localefriendlyName = getLocaleTitle(datasetFieldName,facetFieldCount.getName(), metadataBlockName);
+                    if(metadataBlockName.length() > 0 ) {
```

Comment:
Here and below it looks like we're fixing some whitespace problems. Only three spaces or something.